hbase-server/src/main/java/org/apache/hadoop/hbase/regionserver/HRegion.java

```diff
@@ -912,6 +912,7 @@ long initialize(final CancelableProgressable reporter) throws IOException {
     }
 
     MonitoredTask status = TaskMonitor.get().createStatus("Initializing region " + this);
+    status.enableStatusJournal(true);
```

Comment:
This will produce a journal entry with `null` status like
```
regionserver.HRegion: Region open journal:
null at 1592977559739
Running coprocessor pre-open hook at 1592977559739
Writing region info on filesystem at 1592977559739
...
```

I'm not sure this Is your intention. it looks weird for me. seems better to have meaningful status string.